packages/core/admin/server/tests/admin-api-token.test.e2e.js

```diff
@@ -18,6 +18,10 @@ const { createAuthRequest } = require('../../../../../test/helpers/request');
  * 8. Does not return an error if the ressource to delete does not exist
  * 9. Retrieves a token (successfully)
  * 10. Returns a 404 if the ressource to retrieve does not exist
+ * 11. Updates a token (successfully)
+ * 12. Returns a 404 if the ressource to update does not exist
+ * 13. Fails to creates an api token (missing parameters from the body)
+ * 14. Fails to creates an api token (invalid `type` in the body)
```

Comment:
to create*

Reply:
It's funny. I feel like I had fix that already. I Must have revert the change by mistake maybe 🤷‍♂️

Reply:
I saw that you fixed it in the test, maybe it was that and not in the comment ? 🤷‍♂️

Reply:
That could be it 🤔